apps/avatax/scripts/logger.ts

```diff
@@ -0,0 +1,13 @@
+import { attachLoggerConsoleTransport, createLogger, logger } from "@saleor/apps-logger";
+import { attachLoggerOtelTransport } from "@saleor/apps-logger/node";
+import { loggerContext } from "../src/logger-context";
+
+logger.settings.maskValuesOfKeys = ["username", "password", "token"];
+
+if (process.env.NODE_ENV !== "production") {
+  attachLoggerConsoleTransport(logger);
+}
```

Comment:
We should have env that explicitly enables it

like `__DANGEROUS_ENABLE_MIGRATION_LOGGER`

Reply:
Fixed - right now we have `DANGEROUS_ENABLE_MIGRATION_CONSOLE_LOGGER`